lib/launcher.rb

```diff
@@ -13,18 +15,30 @@ class Launcher
 
     def initialize(settings = SETTINGS)
       @settings = settings
+      @settings.http_server_type = Proxy::SETTINGS.http_server_type.to_sym
```

Comment:
Maybe 
```suggestion
      @settings.http_server_type ||= Proxy::SETTINGS.http_server_type.to_sym
```
We don't want to override user's setting unless the property is not set at all

Reply:
This was just very unintuitive way of converting string to symbol. I have removed this, sticking with strings as the simplified settings in smart proxy does not support any types or anything like that. We use strings everywhere, let's keep that.